content/posts/bayestestR_rnorm_perfect.Rmd

```diff
@@ -0,0 +1,69 @@
+---
+title: "How to easily generate a perfectly normal distribution"
+author: "easystats"
+date: 2019-04-25
+categories: ["R", "bayestestR"]
+tags: ["R", "easystats", "bayestestR", "rnorm", "normal", "gaussian"]
+---
+
+
+```{r message=FALSE, warning=FALSE, include=FALSE}
+library(knitr)
+options(knitr.kable.NA = '')
+```
+
+
+
+Many times, for instance when teaching, I needed to quickly and simply generate a perfectly **perfectly normally distributed sample** to illustrate or show some of its characteristics.
```

Comment:
The word "perfectly" is used twice here

Reply:
Maybe everybody should get that it's really _perfect_! :-D But I also stumbles upon this.

Reply:
How about: "perfectly, perfectly, perfectly"? If two is great, then three must be... perfect (?). That's sound logic right? ;)